rest-server/app/models/physical_venue.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: venues
+#
+#  id          :integer          not null, primary key
+#  name        :text
+#  postal_code :text
+#  prefecture  :text
+#  city        :text
+#  street1     :text
+#  street2     :text
+#  group_id    :integer
+#  url         :text
+#  venue_type  :enum             default(NULL)
+#
+# Indexes
+#
+#  venues_name  (name)
+#
+
+class PhysicalVenue < Venue
+  after_create -> { self.venue_type = :physical }
+  default_scope -> { where(venue_type: 'physical') }
```

Comment:
__[Rails Best Practices]__

default_scope is evil


<!-- Comment by [SideCI](https://sideci.com) -->